Assets/Scripts/Models/Area/World.cs

```diff
@@ -51,6 +51,11 @@ public World(int width, int height, int depth)
         // Creates an empty world.
         SetupWorld(width, height, depth);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
```

Comment:
Yeah you could remove that it is only there so that you can start the game without the main menu scene.